framework/templates/ecl-rating-form/ecl-rating-form.twig

```diff
@@ -0,0 +1,18 @@
+<form class="ma-rating-form">
+  <h3 class="ecl-heading ecl-heading--h3">How useful was this page?</h3>
+  <div class="ecl-u-d-flex">
+    {% for rating in ratings %}
+      <div class="ecl-u-mr-l">
+        {% include '@ec-europa/ema-ratings' with { rating } %}
```

Comment:
For some reasons, the first rating is displayed twice, and not the "average" one.

Reply:
It's because `with { rating }` should be `with rating`